tests/llmobs/conftest.py

```diff
@@ -125,3 +132,10 @@ def AgentlessLLMObs(mock_llmobs_span_agentless_writer, mock_llmobs_eval_metric_w
         llmobs_service.enable(_tracer=dummy_tracer)
         yield llmobs_service
         llmobs_service.disable()
+
+
+@pytest.fixture
```

Comment:
this fixture doesn't do anything right now since `evaluate` is a dummy function anyway that just returns 1.0, but it will set up the testing well for when `evaluate` actually implements a faithfulness evaluation